README.md

```diff
@@ -1,2 +1,88 @@
-# MenuMingle
-MenuMingle is the graduation project of team 6 from the GazaSkyGeeks Code Academy bootcamp.
+# Menu Mingle <img src='./img/chef.png' style='height:30px'>
+our app allows clients to track restaurant orders. It provides an intuitive interface for placing orders, tracking their status, and managing the restaurant's menu.
+## Table of Contents
+- [Introduction](#Menu)
+- [Features](#features)
+- [ UserStory ](#User)
+- [Database Schema](#Database)
+- [Technologies Used](#Technologies)
+- [Demo](#Demo)
+- [Figma Design](#Figma)
+- [Contributing](#contributing)
+- [Acknowledgements](#Acknowledgements)
+# Features 
```

Comment:
Name the features as User Stories, and each story needs to start with `As a user, I should be able to ...`.

E.g.: As a restaurant owner user, I should be able to register and authenticate my staff.